.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,36 @@
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Get version and SHA
+        id: version-sha
+        run: |
+          echo "##[set-output name=version;]$(cat VERSION)"
+          echo "##[set-output name=sha_id;]$(git rev-parse --short HEAD)"
+      - name: Docker login
+        uses: azure/docker-login@v1
+        with:
+          login-server: quay.io
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
I thought we couldn't use secrets directly without adding them with env var ?

Also do you know the benefits of using this azure action while we just do a single command `docker login` ?

Reply:
You can: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#using-encrypted-secrets-in-a-workflow

And `docker login` with `-u <username> -p <password>` is discouraged, I figured it's easier to use a curated GH action for this than doing it ourselves.

Reply:
@skabashnyuk suggested `echo "password" | docker login -u mylogin --password-stdin`

Reply:
IMHO it's a useless action but I don't care